protocol/Protocol.py

```diff
@@ -188,23 +188,16 @@ def versiontuple(version):
 		v+=c
 	return tuple(map(int, (v.split("."))))
 
-def validEngineVersion(engine, version, minver):
-	if engine != "spring":
-		return False
-	if not version:
-		return False
-	return versiontuple(version) >= versiontuple(minver)
 
-	
 flag_map = {
```

Comment:
why the whitespace changes? this breaks layout

Reply:
accident, will fix